src/SFA.DAS.Admin.Roatp.Web/Controllers/AddProvider/SelectProviderController.cs

```diff
@@ -52,11 +56,24 @@ public async Task<IActionResult> Index(AddProviderSubmitModel submitModel, Cance
             return View(viewModel);
         }
 
-        ApiResponse<GetUkrlpResponse> ukrlpResponse = await _outerApiClient.GetUkrlp(int.Parse(submitModel.Ukprn!)!, cancellationToken);
+        ApiResponse<GetUkrlpResponse> ukrlpApiResponse = await _outerApiClient.GetUkrlp(int.Parse(submitModel.Ukprn!)!, cancellationToken);
 
-        if (ukrlpResponse.StatusCode == System.Net.HttpStatusCode.OK)
+        if (ukrlpApiResponse.StatusCode == System.Net.HttpStatusCode.OK)
```

Comment:
Flip this with positive intent... 
`if status code = not found then redirect`
and follow this up with Ensure success status code so any other scenario will throw exception